### DISCUSSION TITLE
点击左上角LOGO返回主页会暴露ip地址和端口号

### DISCUSSION THREAD

**Discussion by JCVICTORSUN** [LIlGG/halo-theme-sakura#413] [General]
### 目前使用的 Sakura 版本？

2.0.0-beta.2

### 目前使用的 halo 版本？

2.7.0

### 建议/问题

是热心的网友发现的

当用域名访问网站后，如果设置左上角为LOGO，点击返回主页会暴露ip地址和端口号
感觉还是挺危险的
希望能快点修复(*╹▽╹*)

**Comment by LIlGG** [comment]
与 Logo 没有太大的关系，此处设置的是你的站点地址，与你的后台设置有关或者创建容器时的 `- --halo.external-url=http://localhost:8090/` 有关

**Comment by JCVICTORSUN** [comment]
okk感谢。这里也提供一下我的解决方法：我自己是用1Panel的运维面板，通过反向代理把网站发布的。所以我在反向代理设置中把后端代理地址的文本替换成了域名。但代价貌似是网站加载会变缓慢）

**Comment by dhe090** [comment]
这个问题我的解决办法是
![image](https://github.com/LIlGG/halo-theme-sakura/assets/47225023/3115e419-aa9b-4b79-bd13-2fc0ee94d4e0)
![image](https://github.com/LIlGG/halo-theme-sakura/assets/47225023/c43859a4-834c-4ba0-a7e5-4f2114e9b6b6)

去1panel里 容器把- --halo.external-url=http://localhost:8090/ 的部分改成你的域名就可以了
